docs/cloud-data-warehouses/snowflake.md

```diff
@@ -1 +1,414 @@
+
+- Day 2 covers the basics of data loading, storage, and querying in Snowflake
+
+## **Day 1**
+
+### What is Snowflake?
+
+Snowflake is a cloud-based data platform that allows organizations to store and analyze large volumes of data. It provides a scalable and flexible solution for data warehousing, enabling users to efficiently manage, share, and query their data using a unique multi-cluster, shared architecture. Snowflake is designed to simplify data management, provide collaboration, and support advanced analytics in a secure and cost-effective manner.
+
+### How Snowflake fits into our project
+
+This is how Snowflake fits into the modern data stack we will be building together:
+- Snowflake is where your data will be stored. We will be gathering all the data for your project and storing it in the RAW tables in your Snowflake account
+- When we get into dbt later on, you’ll see how we use dbt to modify the raw data that is in Snowflake to transform and join data together, to build analyses / dashboards off of
+- Both the raw data and the transformed data (from dbt) will be stored here in Snowflake. 
+
+Some examples of how you might be using Snowflake:
+– If you are an owner of one of the data sources, you’ll need to know what the data coming into raw snowflake should look like, and whether it has landed here correctly. You might set up the data connectors that move data into Snowflake, and you might log into Snowflake to examine and explore the data to ensure the data looks accurate and complete
+
+- If you are a project owner or a systems administrator, you might log into Snowflake to understand storage and query costs, give or revoke access to users, or answer any data security questions about the project.
+  
+- If you are an analytics engineer, you will be doing most of your work to transform the data in dbt, but you should understand that whatever data you query and all the queries you build will actually be running in Snowflake.
+    - You’ll also want to come into Snowflake to look at query history for query optimization
+    - You can explore and take a look at data, column names and types, etc. to better understand how to build the final datasets
+      
+- If you are a data engineer, you might use Snowflake's query history to investigate errors in the data pipeline that dbt can not explain
+  
+- If you are an analyst, you can use Snowflake to run adhoc queries of the data and get instant results. You can use SQL and/or Python to examine the data and look for patterns.
+
+- If you are building reports or dashboards, you will be pulling the data for those directly from Snowflake.
+  
+### Snowflake architecture
+
+Snowflake's architecture is unique in that it separates storage and compute resources. It consists of three layers:
```

Comment:
I'm not sure I would say it's unique, in that lots of big data tools do this! Maybe something like "Snowflake's architecture differs from a traditional transactional database in that..."